entc/gen/func.go

```diff
@@ -108,14 +108,11 @@ func plural(name string) string {
 	return p
 }
 
-// pascal converts the given column name into a PascalCase.
-//
-//	user_info => UserInfo
-//	full_name => FullName
-//	user_id   => UserID
-//
-func pascal(s string) string {
-	words := strings.Split(s, "_")
+func isSeparatorRune(r rune) bool {
```

Comment:
Don't need to add the type to the func name. Let's remove it.

```suggestion
func isSeparator(r rune) bool {
```

Reply:
Great catch - updated